modules/culling/src/lib/oriented-bounding-box.d.ts

```diff
@@ -17,8 +17,14 @@ export default class OrientedBoundingBox {
   // An OrientedBoundingBox of some object is a closed and convex cuboid. It can provide a tighter bounding volume than {@link BoundingSphere} or {@link AxisAlignedBoundingBox} in many cases.
   constructor(center?: readonly number[], halfAxes?: readonly number[]);
 
+  // Returns halfSize array corresponding to I3S OBB halfSize (https://github.com/Esri/i3s-spec/blob/master/docs/1.7/obb.cmn.md)
```

Comment:
If you use JSDoc style comments, I think might become visible in vscode when hovering:
`/** */`

Reply:
Do you mean hypertext? The link works in vscode as is now.